mmv1/templates/terraform/custom_flatten/dialogflowcx_agent_git_integration_settings_github_settings.go.erb

```diff
@@ -0,0 +1,31 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2018 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func flattenDialogflowCXAgentGitIntegrationSettingsGithubSettings(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	if v == nil {
+		return nil
+	}
+	original := v.(map[string]interface{})
+	if len(original) < 2 {
+		// original has only access_token
+		return nil
+	}
```

Comment:
Looking at the nightly test failures - it looks like even if we don't set _anything_ for gitIntegrationSettings, the API will return a block like:

```
  "gitIntegrationSettings": {
    "githubSettings": {
      "accessToken": "REDACTED"
    }
  }
```

I'm a little concerned about checking just for `< 2` - if the API later _stops_ returning `accessToken: REDACTED` that could end up breaking things. It's also not immediately clear from the implementation _why_ we're using `< 2` - or more specifically, why this flatten function is behaving differently than the standard.

What would y'all think about instead explicitly removing accessToken and then going back to standard behavior - something like:

```suggestion
	// ignore access_token, which is always returned as REDACTED
	delete(original, "access_token");
	if len(original) == 0 {
		return nil
	}
```